src/validation-group.js

```diff
@@ -408,16 +408,16 @@ export class ValidationGroup {
    * @param threshold {Object} An optional threshold that will be passed to the customFunction
    * @returns {ValidationGroup} returns this ValidationGroup, to enable fluent API
    */
-  passes(customFunction, threshold) {
+  passes(customFunction: Function, threshold: any): ValidationGroup {
```

Comment:
Done it as suggested, I've tried to look up doc and also look into code however cant determine the actual types. Better leave them any for now. If I happen to use them at a later stage I will update.